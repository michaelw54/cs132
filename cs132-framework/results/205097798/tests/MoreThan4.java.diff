1,4c1,13
< ParseException: Encountered " <IDENTIFIER> "w_2 "" at line 2, column 1.
< Was expecting:
<     "return" ...
<     
---
> 1
> 2
> 3
> 4
> 5
> 6
> 6
> 5
> 4
> 3
> 2
> 1
> 0
