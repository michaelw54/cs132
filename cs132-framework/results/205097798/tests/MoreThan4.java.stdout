func Main()
w_2 = alloc(4)
w_3 = alloc(8)
[w_2 + 0] = w_3
[w_3 + 0] = @MT4_Start
[w_3 + 4] = @MT4_Change
w_2 = 1
w_3 = 2
w_4 = 3
w_5 = 4
w_6 = 5
w_7 = 6
w_1 = @MT4_Start
w_0 = call w_1(w_2)
print(w_0)
ret_ = 0
return ret_

Type error
